### PR TITLE
P2022-1134 - components success and error alerts

### PR DESCRIPTION
**Pull request by alicjakarczewska**
https://tracker.intive.com/jira/browse/P2022-1134

I added components AlertSucess, AlertError (they differ in display time and style [two options from MUI]), and two more components (ButtonShowingAlertSuccess, ButtonShowingAlertError) to show how to use them and how do they work. I also added their stories to the storybook.

### CODE REVIEW

File: src/components/AlertError/AlertError.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+export default function AlertError({
+  alertMsg,
+  isOpen,
+  handleClose,
+}: AlertErrorProps) {
+  return (
+    <Snackbar open={isOpen} onClose={handleClose}>
+      <Alert onClose={handleClose} severity='error' sx={{ width: "100%" }}>
```
Comment (jprymak) [inline src/components/AlertError/AlertError.tsx:25]: It doesn't seem like this sx prop does anything. We also agreed not to use sx if possible ;)

File: src/components/AlertSuccess/AlertSuccess.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+export default function AlertSuccess({
+  alertMsg,
+  isOpen,
+  handleClose,
+}: AlertSuccessProps) {
+  return (
+    <Snackbar open={isOpen} autoHideDuration={3000} onClose={handleClose}>
+      <Alert onClose={handleClose} severity='success' sx={{ width: "100%" }}>
```
Comment (jprymak) [inline src/components/AlertSuccess/AlertSuccess.tsx:25]: It doesn't seem like this sx prop does anything. We also agreed not to use sx if possible ;)

**Comment by jprymak** [comment]
AlertSuccess and AlertError components are almost the same. If I were you, I would try to create a component that takes a variant - "success" or "error" as a prop and displays result accordingly

File: src/components/AlertError/AlertError.tsx (hunk 1 above)
Comment (alicjakarczewska) [reply src/components/AlertError/AlertError.tsx]: Ok, I copied it from MUI but then I removed this property

File: src/components/AlertSuccess/AlertSuccess.tsx (hunk 2 above)
Comment (alicjakarczewska) [reply src/components/AlertSuccess/AlertSuccess.tsx]: Ok, I copied it from MUI but then I removed this property

**Review by jprymak — APPROVED** [review]
Nice

**Pull request merged**